### PR TITLE
Improve CSS

### PR DESCRIPTION
**Pull request by cindyli**
Your "design-impl" that https://github.com/inclusive-design/covid-data-monitor/pull/5 is based off has already been merged into this branch. All conflicts have been resolved.

### CODE REVIEW

File: src/css/fluid-covid-map-viz.css [hunk 1]
```diff
@@ -379,3 +452,95 @@ body::after {
...
+
+    /******** The panel for displaying search results ********/
+    .fl-mapviz-search-results {
+        height: 36rem;
+        margin-right: 1rem;
+    }
+
+    .fl-mapviz-back-button {
+        justify-content: left;
+        padding-left: 1rem;
```
Comment (amb26) [inline src/css/fluid-covid-map-viz.css]: I found that this padding definition is causing a misalignment of the entire button block - together with the "width: 100%" directive of the container, it causes the back button to stick out by 1 rem into the map

File: index.html [hunk 2]
```diff
@@ -83,136 +84,203 @@ <h2>Accessibility Map for COVID-19 assessment centres</h2>
...
-                            <span class="fl-mapviz-filter-icon"></span>
-                            <span class="fl-mapviz-filter-tooltip-icon fl-mapviz-hoverable"></span>
-                            <span class="fl-mapviz-filter-tooltip"></span>
-                            <label class="fl-mapviz-filter-title" for="toggle-2">Accessible Washrooms</label>
-                            <label class="fl-checkbox-holder" tabindex="0">
-                                <input class="fl-mapviz-checkbox visually-hidden" tabindex="-1" id="toggle-2" type="checkbox" value="true" />
-                                <span></span>
-                            </label>
+                        <div class="fl-mapviz-filter-buttons">
+                            <button class="fl-mapviz-reset-filters" type="button">
```
Comment (amb26) [inline index.html]: The loss of the class fl-mapviz-hoverable here has removed the hover effect

File: index.html [hunk 3]
```diff
@@ -83,136 +84,203 @@ <h2>Accessibility Map for COVID-19 assessment centres</h2>
...
-                        <div class="fl-mapviz-filter fl-mapviz-filter-washrooms">
-                            <span class="fl-mapviz-filter-icon"></span>
-                            <span class="fl-mapviz-filter-tooltip-icon fl-mapviz-hoverable"></span>
-                            <span class="fl-mapviz-filter-tooltip"></span>
-                            <label class="fl-mapviz-filter-title" for="toggle-2">Accessible Washrooms</label>
-                            <label class="fl-checkbox-holder" tabindex="0">
-                                <input class="fl-mapviz-checkbox visually-hidden" tabindex="-1" id="toggle-2" type="checkbox" value="true" />
-                                <span></span>
-                            </label>
+                        <div class="fl-mapviz-filter-buttons">
```
Comment (amb26) [inline index.html:142]: The change in structure of the markup here has caused the tab focus ring to be strangely misshapen - when keyboard focus is on this button, the bottom part of the ring is now jagged

File: index.html [hunk 4]
```diff
@@ -83,136 +84,203 @@ <h2>Accessibility Map for COVID-19 assessment centres</h2>
...
+                    <!-- A11y filter panel -->
+                    <div class="fl-mapviz-filter-panel">
+                        <div class="fl-mapviz-filters">
+                           <div class="fl-mapviz-filter fl-mapviz-filter-entrances">
+                               <span class="fl-mapviz-filter-icon"></span>
+                               <span class="fl-mapviz-filter-tooltip-icon flc-mapviz-hoverable"></span>
+                               <span class="fl-mapviz-filter-tooltip"></span>
+                               <label class="fl-mapviz-filter-title" for="toggle-1">Accessible Entrances</label>
+                               <label class="fl-checkbox-holder" tabindex="0">
+                                   <input class="fl-mapviz-checkbox visually-hidden" id="toggle-1" type="checkbox"/>
```
Comment (amb26) [inline index.html]: The removal of tabindex=-1 from these elements has caused them to become spuriously keyboard focusable, but no focus indicator appears. The attribute should be put back

**Comment by cindyli** [comment]
Thanks for the review, @amb26. All comments have been addressed.

Tomorrow I will give a try to use CSS to add circle shadows for filter button icons and back icon.

**Pull request merged**